### ISSUE TITLE
can we provide a single unified version number for fabric8?

### ISSUE THREAD

**Issue by jstrachan** [fabric8io/fabric8#6338]
right now its too confusing, there's about 5 independent version numbers of 'fabric8'. It makes things challenging on the changelog and for folks to keep track of regressions and so forth.

I wonder if whenever we release anything, we should release 'fabric8 microservices platform' which is really just the unified set of apps we install.

Then as we install we can say 'this is version 1.x.y of fabric8 microservices platform' then users can remember that (we can put it into the configmap somewhere). Then if one version doesn't work too well folks can just revert to the one version they know.

Otherwise there's so many permutations of versions of things that it could get complex.

**Comment by chenyanxu** [comment]
I suggest if we can also provide a change log file, then we will know which version had changed.

**Comment by jstrachan** [comment]
@chenyanxu definitely! I also want a status.fabric8.io page so if we find new releases of sme permutation of: minikube / minishift / openshift / kubernetes / gofabric8 / fabric8 breaks something, we can list known permutations that have issues & list workarounds. There's so many projects + places issues get raised it might be simplest to have a single page
